lib/firespring_dev_commands/git.rb

```diff
@@ -185,7 +185,7 @@ def reset(dir: default_project_dir)
     end
 
     # Checks out the given branch in all repositories with some additional formatting
-    def checkout_all(branch)
+    def checkout_all(branch, default_branch: nil)
```

Comment:
Are these nil values necessary? I believe you could just use one variable through all of the calls as the return of the .to_s.strip would return nil on empty.